lib/xhr.js

```diff
@@ -322,7 +322,7 @@ function enableXHROverH2(XMLHttpRequest, configuration) {
         var self = this;
 
         // Also fix support for node-http2 FormData
-        body = serializeXhrBody(xhrInfo, body);
```

Comment:
Please move with or remove corresponding comment 
// Also fix support for node-http2 FormData